cmd/launch.go

```diff
@@ -61,6 +63,10 @@ func init() {
 		"The auto-termination timer for the instance in minutes")
 	launchCmd.Flags().StringVarP(&flagConfig.IamInstanceProfile, "iam-instance-profile", "p", "",
 		"The profile containing an IAM role to attach to the instance")
+	launchCmd.Flags().StringVarP(&flagConfig.UserDataFilePath, "boot-script", "b", "",
+		"The absolute filepath to a bash script passed to the instance and executed after the instance starts (user data)")
+	launchCmd.Flags().StringSliceVar(&flagConfig.UserTags, "tags", nil,
```

Comment:
why not use `StringToStringVar` to read a map? 

That way, 
- `:` can be a valid char in a tag value 
- validating with `strings.Split(rawTag, ":")` and length is exactly 2 won't be necessary.

Reply:
this is what I was looking for! I'll update in next revision